local-volume/provisioner/deployment/docker/Dockerfile

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM scratch
+FROM busybox
```

Comment:
@msau42 I used busybox because it allows a certain level of debug-ability as you can ssh into the container and use some basic linux utils to see what the container is viewing on a host. Seems useful for a service that is yet to mature. I can remove it if thats not something we want.